homeassistant/components/somfy/cover.py

```diff
@@ -88,6 +138,13 @@ def is_closed(self):
             is_closed = self._closed
         return is_closed
 
+    @property
+    def assumed_state(self):
```

Comment:
You can move it within somfy device class.

Reply:
done